packages/plugin-core/src/commands/GotoNote.ts

```diff
@@ -62,7 +62,7 @@ export class GotoNoteCommand extends BasicCommand<
   GoToNoteCommandOutput
 > {
   key = DENDRON_COMMANDS.GOTO_NOTE.key;
-  private extension: IDendronExtension;
+  protected extension: IDendronExtension;
```

Comment:
Probably makes most sense to remove this field of GoToCommand level now and use the BaseCommand.extension, otherwise I believe we are hiding the BaseCommands field and that becomes unintuitive maintainance wise.